app/views/users/dashboard.html.erb

```diff
@@ -7,31 +7,74 @@
-                <label class="btn btn-default active">
-                  <input type="checkbox" autocomplete="on" checked><%=category.name %>
-                </label>
-                <% end %>
-              </div>
-              <label>Sort by:</label>
-              <div class="btn-group btn-group-sm" data-toggle="buttons">
-                <label class="btn btn-default active">
-                  <input type="radio" name="options" autocomplete="off" checked> Title
-                </label>
-                <label class="btn btn-default">
-                  <input type="radio" name="options" autocomplete="off"> Author
-                </label>
-              </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="wrapper wrapper--light">
+    <div class="container">
+      <div class="pagehead">
+        <div class="row">
+          <div class="col-xs-12">
+            <label>Current Decks</label>
+            <div class="grid">
+              <% @decks.take(10).each_with_index do |deck, index| %>
+                <%= render 'dashboard_current_deck', deck: deck, index: index %>
+              <% end %>
             </div>
           </div>
         </div>
       </div>
     </div>
   </div>
-  <div class="container">
+  <div class="container browse-decks-container">
+    <div class="row">
+      <div class="col-xs-12">
+        <label>Browse Decks</label>
+        <div class="btn-toolbar pull-right">
+          <div class="btn-group">
+            <button 
+              type="button"
+              class="btn btn-light btn-sm btn-category-filter"
+              data-container=".browse-decks-container"
+              data-role="filter"
+              data-toggle="popover"
+              data-placement="bottom"
+              data-animation="false"
+              data-html="true"
+              data-original-title="<span class='btn-close-category-filter'>
+                                     Categories 
+                                     <span class='dropup'>
+                                       <span class='caret'></span>
+                                     </span>
+                                   </span>"
+              data-content="<div class='filter-group'>
+                              <% Category.all.each do |category| %>
+                                <label class='btn btn-sm btn-light'>
+                                  <input type='checkbox' autocomplete='on' checked><%= category.name %>
+                                </label>
+                              <% end %>
+                            </div>
+                            <div class='popover-footer'>
+                              <a href='#' class='btn btn-link'>
+                                <span class='glyphicon glyphicon-refresh'></span> Refresh Categories
+                              </a>
+                            </div>">
+              Categories <span class="caret"></span>
```

Comment:
I saw your note about getting the checkboxes to reflect the current filter. To do it with rails I'd have to play around with it, but with javascript you could parse out the URL and update the checkboxes accordingly (not that i love that solution but it'd probably work).

However, more importantly, I'm not sure this is the best UI for this task. It's flexible in that a user can filter by multiple categories, which is cool. But that might even be overkill for what we need here. I think all we _need_ is the ability to filter by ` category at a time, which would be quite a bit easier to build out. Take FiveThirtyEight for example (or basically any other news site)
<img width="840" alt="screen shot 2016-04-29 at 5 12 34 pm" src="https://cloud.githubusercontent.com/assets/1417199/14931500/8d61dc80-0e31-11e6-8159-a15711399842.png">
Where they have Politics, Sports, Science & Health... We might have categories listed out such as Philosophy, Economics, History, etc. Each link could filter for just that category, much like the functionality already existing on the admin dashboard. Thoughts?